.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish to FTP on Development Push
```

Comment:
change the name of the fail to something more descriptive. What is this yaml file for?
maybe something like deploy_plan?